src/main/java/ch/njol/skript/expressions/base/EventValueExpression.java

```diff
@@ -57,80 +57,42 @@
  * }
  * </pre>
  * 
- * @author Peter Güttinger
  * @see Classes#registerClass(ClassInfo)
  * @see ClassInfo#defaultExpression(DefaultExpression)
  * @see DefaultExpression
  */
 public class EventValueExpression<T> extends SimpleExpression<T> implements DefaultExpression<T> {
-	
-	private final Class<? extends T> c;
+
+	private final Map<Class<? extends Event>, Converter<?, ? extends T>> converters = new HashMap<>();
```

Comment:
This likely needs to be a ConcurrentHashMap for when parallel-loading is properly supported in the future